src/app/zones/components/ZoneCreateKubernetesInstructions.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div>
-    <h3>1. {{ i18n.t('zones.form.kubernetes.prerequisites.title') }}</h3>
+    <h3>
+      <span class="step-number">1</span>
```

Comment:
Probably not for this PR, but it feels like this should be an order list instead of headers?

Reply:
Not sure if there’s much gained of markup up a big list of instructions as an `ol`, but I feel indifferent about it so we could change this.

Reply:
Yeah sure, not for this PR. But more generally, if you are keen on avoiding things like `[role=button]` then we should be keen on doing things like this also.